analysis_of_products_in_stores/Task_2_1.py

```diff
@@ -0,0 +1,142 @@
+import os
+import csv
+from itertools import groupby
+from collections import Counter
+from common_functions.work_with_document import save_file
+
+def get_data(filename):
+    """function returns a dictionary from the datafile"""
+
+    ROOT_DIR = os.path.dirname(os.path.abspath(__file__))
+    with open(os.path.join(ROOT_DIR, filename)) as file:
+        reader = csv.reader(file)
+        next(reader)
+        main_dict = [
+            {
+                "ITEM_ID": row[0],
+                "STORE_ID": row[1],
+                "PRICE": float(row[2]),
+                "APPROVED_BY": row[3],
+            }
+            for row in reader
+        ]
+    return main_dict
+
+def sort_dict(dict, field):
+    """function returns a sorted dictionary for a field which was given like function's parameter and this parameter"""
+
+    sorted_dict = sorted(dict, key=lambda position: position[field])
+    return sorted_dict, field
+
+def group_dict(def_sort_dict):
+    """function returns a grouped dictionary"""
+
+    grouped_dict = {
+        key: [group for group in groups]
+        for key, groups
+        in groupby(def_sort_dict[0], key=lambda position: position[def_sort_dict[1]])
+    }
+    return grouped_dict
+
+def count_unique_values(dict):
```

Comment:
oh, exactly. Done.

Reply:
Let's get rid of this function at all, it doesn't seems to be useful, because it just call len(dict). Creating a function for the sake of creating the function is doubtful.

Reply:
Done